doc/source/examples/dem/bunny-drill/bunny-drill.rst

```diff
@@ -0,0 +1,207 @@
+==================================
+Bunny Drill
+==================================
+
+This example simulates the drilling motion of a bunny within a bed of particles. It illustrates how the DEM module of Lethe can simulate complex moving objects and is a testament to our love of lagomorphs. Do not worry friend, no bunnies were hurt in the making of this example!
+
+
+----------------------------------
+Features
+----------------------------------
+- Solvers: ``dem``
+- Floating walls
+- `GMSH <https://gmsh.info/>`_ grids
+
+----------------------------
+Files Used in This Example
+----------------------------
+- Parameter file used to load the particles: ``examples/dem/3d-bunny-drill/bunny-drill-loading.prm``
+- Parameter file used to simulate the bunny drill: ``examples/dem/3d-bunny-drill/bunny-drill.prm``
+- GMSH mesh of the bunny: ``examples/dem/3d-bunny-drill/bunny-drill.prm``
+
+-----------------------
+Description of the Case
+-----------------------
+
+This simulation consists of two stages: filling (0-2 s) and bunny-drilling (2-4.75 s) of particles.
+
+--------------
+Parameter File
+--------------
+
+Mesh
+~~~~~
+
+The mesh is a cylinder generated using the deal.II grid generator.
+
+.. code-block:: text
+
+  subsection mesh
+    set type               = cylinder
+    set grid type          = balanced
+    set grid arguments     = 6 : 0.10 : 0.25
+    set initial refinement = 2
+  end
+
+Insertion Info
+~~~~~~~~~~~~~~~~~~~
+
+An insertion box is defined inside and the top part of the cylinder. The insertion region is sufficiently high to ensure that there are no collisiosn between the particles bouncing on the bottom wall (or on particles accumulated at the bottom) and the ones being inserted.
```

Comment:
```suggestion
An insertion box is defined at the top part of the cylinder. The insertion region is sufficiently high to ensure that there are no collisions between the particles bouncing on the bottom wall (or on particles accumulated at the bottom) and the ones being inserted.
```

Reply:
"guarantee that particles are not inserted at another particle's position" or "avoid overlapping at insertion"?